crates/aptos/src/common/init.rs

```diff
@@ -159,34 +158,21 @@ impl CliCommand<()> for InitTool {
         };
         let public_key = private_key.public_key();
 
+        let client = aptos_rest_client::Client::new(
+            Url::parse(profile_config.rest_url.as_ref().unwrap())
+                .map_err(|err| CliError::UnableToParse("rest_url", err.to_string()))?,
+        );
+
         // lookup the address from onchain instead of deriving it
         // if this is the rotated key, deriving it will outputs an incorrect address
```

Comment:
This comment feels out of date since it doesn't cover what happens if the address can't be found on chain.